crates/essential-rest-client/src/builder_client.rs

```diff
@@ -41,8 +41,8 @@ impl EssentialBuilderClient {
                 .iter()
                 .map(|p| register_program_solution(big_bang.program_registry.clone(), p)),
         );
-        let solutions = SolutionSet { solutions };
-        self.submit_solution(&solutions).await
+        let solution_set = SolutionSet { solutions };
+        self.submit_solution_set(&solution_set).await
     }
 
     /// Submit solution.
```

Comment:
```suggestion
    /// Submit solution set.
```

Suggesting the same change for the rest of this doc comments.